entity/src/main/java/com/wilma/entity/positions/ExpressionOfInterest.java

```diff
@@ -0,0 +1,46 @@
+package com.wilma.entity.positions;
+
+import javax.persistence.Entity;
+
+import java.time.Period;
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Table;
+
+import com.wilma.entity.users.Partner;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+
+@Getter
+@Setter
+@Entity
+@NoArgsConstructor
+@Table(name = "expression_of_interest")
+public class ExpressionOfInterest extends PositionCategory {
```

Comment:
An expression of interest is a category?

Reply:
No it shouldn't be - hence my confusion and asking for team input into what we should do.